src/ol/DataTile.js

```diff
@@ -6,7 +6,7 @@ import TileState from './TileState.js';
 import {createCanvasContext2D} from './dom.js';
 
 /**
- * @typedef {HTMLImageElement|HTMLCanvasElement|HTMLVideoElement} ImageLike
+ * @typedef {HTMLImageElement|HTMLCanvasElement|HTMLVideoElement|ImageBitmap} ImageLike
```

Comment:
`ImageBitmap` works with the existing data tiles, and has the advantage of being decoded already - if created properly right after `HTMLImageElement#decode()`.